store/sqlstore/team_store.go

```diff
@@ -1397,8 +1398,10 @@ func (s SqlTeamStore) ResetAllTeamSchemes() error {
 // ClearCaches method not implemented.
 func (s SqlTeamStore) ClearCaches() {}
 
+// InvalidateAllTeamIdsForUser method not implemented
```

Comment:
This method is there for satisfy the interface, but is handled by the cache layer that is build on top of this. Maybe the test is "is not executing anything because the store is not handling the cache".

Reply:
Sounds good :+1: